test/helpers/helpers.js

```diff
@@ -37,6 +38,21 @@ beforeEach(() => {
           return result;
         }
       };
-    }
+    },
+    toDeepEqual: () => ({
+      compare: async(spec, expected) => {
+        const result = {};
+        const actual = marked(spec.markdown, spec.options);
+        result.pass = assert.deepStrictEqual(expected, actual) === undefined;
```

Comment:
I feel like just using `===` would be good enough since we know both will always be strings.

```suggestion
        result.pass = expected === actual;
```